pennylane/operation.py

```diff
@@ -422,12 +422,6 @@ def __init__(self, angle, wire_rot, wire_flip=None, do_flip=False,
                 if do_flip and wire_flip is None:
                     raise ValueError("Expected a wire to flip; got None.")
 
-                # note: we use the framework-agnostic math library since
-                # trainable inputs could be tensors of different types
-                shape = qml.math.shape(angle)
-                if len(shape) > 1:
-                    raise ValueError(f"Expected a scalar angle; got angle of shape {shape}.")
```

Comment:
How come this was removed?

Reply:
With introducing the `ndim_params` argument, there come tests that are run at initialization, which include testing the shape of the input. This makes this manual test unnecessary, and I felt we should not educate users to do redundant tests :)